safetrace-app/Features/Contact Tracing/ContactTracingViewController.swift

```diff
@@ -373,43 +373,50 @@ class ContactTracingViewController: UIViewController {
 
         let privacyImageLabelView = UIStackView(arrangedSubviews: [privacyIcon, privacyTextView])
         update(privacyImageLabelView, ContactTracingStyle.imageLabelStackView)
+        
+        let versionLabel = makeAppVersionLabel()
```

Comment:
Nit: Can we not include it in the disabled state, or at least increase the spacing to the text above? The version number clashes with the permissions rows stylistically.

Reply:
Sure, added some spacing